src/components/ec-table-filter/ec-table-filter.story.js

```diff
@@ -280,6 +308,20 @@ const AllTemplate = storyArgs => ({
         />
       </div>
     </div>
+    <h2 class="tw-m-24">Use custom CSS Grid layout</h2>
+    <div class="tw-flex tw-px-20">
+      <div class="tw-my-auto tw-mx-20 tw-w-full ec-card">
+        <ec-table-filter
+          v-model="model"
+          :filters="searchFilterFillingRemainingSpace"
+          layout-class="!tw-grid tw-grid-rows-2 !tw-grid-flow-row tw-grid-cols-[repeat(4,_auto)_1fr_auto] lg:tw-grid-rows-1"
```

Comment:
BTW, we should never use `important`. Why is that needed?

Reply:
This is only needed here because TW classes have lower specificity than the CSS classes we used inside the component, so styles are not being applied (unless we wrapped those in a CSS class, but I don't know if we can do that in Storybook stories tbh)

Reply:
We should use BEM and only have classes with specificity of 0100 so they never clash and can be easily overridden. That's the whole purpose of using BEM.